pkg/cmd/repo/setdefault/setdefault_test.go

```diff
@@ -499,6 +504,7 @@ func TestDefaultRun(t *testing.T) {
 			}
 			assert.NoError(t, err)
 			assert.Equal(t, tt.wantStdout, stdout.String())
+			assert.Equal(t, tt.wantStderr, stderr.String())
```

Comment:
Last thing here: I think it'd be helpful to separate these with a conditional:

```suggestion
			if tt.wantStdout != "" {
			    assert.Equal(t, tt.wantStdout, stdout.String())
			} else {
			    assert.Equal(t, tt.wantStderr, stderr.String())
			}
```

This allows us to get only one string matching assert error per test case, meaning it's easier to understand what in the test is failing (`Stderr` or `Stdout`). Right now, we're getting two error outputs that are mirrors of one another, making understanding which part of the code is failing a bit more difficult.